assets/model_monitoring/components/src/shared_utilities/histogram_utils.py

```diff
@@ -43,6 +43,12 @@ def get_dual_histogram_bin_edges(
 
         bin_width = (max_value - min_value) / num_bins
 
+        # If histogram has only one value then we only need a single bucket and
+        # should skip the for-loop below.
+        if min_value == max_value:
+            all_bin_edges[col] = [min_value, min_value * 1.01]
```

Comment:
Also note that there is an edge-case for the edge-case. This approach will not work if `min_value==0`, as the bin_edges would remain [0,0]. If you use the `delta` approach above, you can check whether `delta==0`, in which case you force `delta=0.005` or another arbitrary number.